### PR TITLE
Change most ednotes to issue markers, and remove some closed issues.

### PR DESCRIPTION
**Pull request by gkellogg**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/pull/115.html" title="Last updated on Jun 9, 2023, 12:05 AM UTC (16c9f14)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-canon/115/e98a348...16c9f14.html" title="Last updated on Jun 9, 2023, 12:05 AM UTC (16c9f14)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -562,15 +547,12 @@ <h2>Blank Node Identifier Issuer State</h2>
     <h2>Canonicalization Algorithm</h2>
 
     <p class="ednote">At the time of writing, there are several open issues that will determine important details of the canonicalization algorithm.</p>
-    <div class="issue" data-number="7"></div>
-    <div class="issue" data-number="8"></div>
     <div class="issue" data-number="10"></div>
     <div class="issue" data-number="11"></div>
-    <div class="issue" data-number="16"></div>
     <div class="issue" data-number="84"></div>
```
Comment (yamdan) [inline spec/index.html:552]:
```suggestion
```
We can also remove the marker for #84 as it has already been closed.

File: spec/index.html [hunk 2]
```diff
@@ -465,9 +452,7 @@ <h2>Canonicalization</h2>
     "Universal RDF Dataset Canonicalization Algorithm 2015"
     (<abbr title="Universal RDF Dataset Canonicalization Algorithm 2015"><dfn class="export">URDNA2015</dfn></abbr>).</p>
```
Comment (yamdan) [inline spec/index.html:454]:
```suggestion

    <div class="issue" data-number="87"></div>
    <div class="issue" data-number="88"></div>
```
Not hugely important, but as both #87 and #88 pertain to the name of the algorithm, I thought they would be better placed here.

File: spec/index.html [hunk 3]
```diff
@@ -562,15 +547,12 @@ <h2>Blank Node Identifier Issuer State</h2>
...
 
     <p class="ednote">At the time of writing, there are several open issues that will determine important details of the canonicalization algorithm.</p>
-    <div class="issue" data-number="7"></div>
-    <div class="issue" data-number="8"></div>
     <div class="issue" data-number="10"></div>
     <div class="issue" data-number="11"></div>
-    <div class="issue" data-number="16"></div>
     <div class="issue" data-number="84"></div>
     <div class="issue" data-number="87"></div>
     <div class="issue" data-number="88"></div>
```
Comment (yamdan) [inline spec/index.html:554]:
```suggestion
```
Not hugely important, but as both #87 and #88 pertain to the name of the algorithm, I thought they would be better placed above.

**Review by yamdan — APPROVED** [review]
Thank you @gkellogg , I just added a few comments.

File: spec/index.html (hunk 2 above)
Comment (gkellogg) [reply spec/index.html:454]: They’ll be removed in #116.

**Review by iherman — APPROVED** [review]
I believe that "proposed closing" marked issues, as well as issue 87 and 88 should be removed, too.

**Review by dlongley — APPROVED** [review]

**Pull request merged**